src/mail.sh

```diff
@@ -63,10 +67,42 @@ function validate_encryption()
   fi
 }
 
+# This function checks that the necessary options to use
```

Comment:
What do you mean by `necessary`? What are the conditions? Please, improve this comment.